bridges/snowbridge/primitives/core/src/operating_mode.rs

```diff
@@ -23,3 +23,7 @@ impl BasicOperatingMode {
 		*self == BasicOperatingMode::Halted
 	}
 }
+
+pub trait IsHalted {
+	fn is_halted() -> bool;
+}
```

Comment:
Naming is too generic.

Also needs a documentation.

```suggestion
pub trait ExportPausedQuery {
	fn is_paused() -> bool;
}
```